components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,278 @@
+✓ Created origin effortless.
+75 B / 75 B | [===========================================] 100.00 % 1.31 MB/s d
+» Uploading public origin key chef-20160614114050.pub
+...
+```
+
+The `--auto-create-origins` flag creates each origin listed in the
+`<artifact-dir>/artifacts` directory. If you omit the `--auto-create-origins` flag,
+use the Chef Habitat Builder UI to create the necessary origins before running the
+`bulkupload` command.
+
+To finish up, return to your Chef Habitat Builder on-prem installation and view the packages that you've added to your `core` origin at `https://chef-automate.test/bldr/#/origins/core/packages`.
+
+## Operating Chef Habitat Builder
+
+Chef Habitat Builder uses the same mechanisms that Chef Automate does for [backups]({{< relref "backup.md" >}}), [log management]({{< relref "log-management.md" >}}), and [uninstalling]({{< relref "troubleshooting.md#uninstalling-chef-automate" >}}).
+
+To change the log level for Chef Habitat Builder only, create a TOML file that contains the partial configuration below. Uncomment and change settings as needed, then run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
+
+```toml
+[builder_api.v1.sys.log]
+level = "debug"
+scoped_levels = ["tokio_core=error", "tokio_reactor=error", "zmq=error", "hyper=error" ]
+```
+
+## Setting up Automate as an OAuth Provider for Habitat Builder (Deprecated)
+
+{{< warning >}}
+These instructions have been deprecated in favor of using the Chef Automate installer to deploy Chef Habitat on-prem
+{{< /warning >}}
+
+To configure Chef Automate as an OAuth Provider for Habitat Builder, create a TOML file with the partial configuration below.
+Run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
+
+`bldr_client_id` and `bldr_client_secret` simply need to match what you configured for the corresponding
+values in Habitat Builder (see below). However, we strongly recommend those values follow
+[best practices](https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/)
+for `client_id` and `client_secret` in the Oauth2 standard.
+
+```toml
+[session.v1.sys.service]
+bldr_signin_url = "<your Builder fqdn>" # for example, "http://builder.test/"
+
+# This needs to match what you configured OAUTH_CLIENT_ID as when you configured Habitat Builder.
+bldr_client_id = "<your Habitat Builder Oauth2 Client ID>"
+
+# This needs to match what you configured OAUTH_CLIENT_SECRET as when you configured Habitat Builder.
+bldr_client_secret = "<your Habitat Builder Oauth2 Client Secret>"
+```
+
+You'll need to add Automate's TLS certificate to Builder's list of accepted certificates
+in addition to these configuration changes.
+Locate Automate's default self-signed certificate by running `cat /hab/svc/automate-load-balancer/data/{{< example_fqdn "automate" >}}.cert`
+You can copy this default certificate, and then add it to your Builder instance's list of accepted certs.
```

Comment:
```suggestion
```

Reply:
@IanMadd why are you suggesting that we remove this line?